samples/client/petstore/java/feign-no-nullable/src/main/java/org/openapitools/client/ApiClient.java

```diff
@@ -123,6 +123,16 @@ private ObjectMapper createObjectMapper() {
     return objectMapper;
   }
 
+  private RequestInterceptor buildOauthRequestInterceptor(OAuthFlow flow, String authorizationUrl, String tokenUrl, String scopes) {
```

Comment:
This change appeared when I generated the samples, but it is not related to my code changes.

Reply:
Yes, we've updated that in the master.